### ISSUE TITLE
Tips for management: how accessibility testing actually impacts your development process

### ISSUE THREAD

**Issue by ljgoslin** [a11yproject/a11yproject.com#715]
[Join our live webinar](https://accessibility.deque.com/tips-for-management-how-accessibility-testing-actually-impacts-your-development-process) to learn what accessibility testing actually looks like in your organization and development.

Tuesday, January 17th from 1:00-2:00PM EST

This webinar is geared towards those who have been recently tasked with accessibility, are starting to think about how to automate accessibility testing, or are curious about different ways to implement testing into their organization. Noah Mashni, Solutions Architect at Deque, will answer common questions people have when they’re curious about how accessibility will impact their current organization and developers.

In this webinar, you will learn:

- How accessibility testing solutions should be agnostic to your existing architecture
- How to build sustainable accessibility, i.e. how accessibility can be part of your whole development process without interrupting or breaking your builds.
- Where accessibility lives inside your project or firewall
- How you can automate accessibility for every point of your accessibility journey. Whether you’re organization is using testing frameworks or is new to automated testing, accessibility can fit into the early stages of development.

**Comment by ericwbailey** [comment]
Closed via https://github.com/a11yproject/a11yproject.com/commit/e45ee7f1e41959b933df0070a0c9d2e3b57dcbda.

**Comment by ljgoslin** [comment]
Hey Eric, this is on January 17th - looks like it is showing under past events. Could you please update? Thanks!

Cheers - Laura

**Comment by ericwbailey** [comment]
My bad, sorry. I'll get to this and https://github.com/a11yproject/a11yproject.com/issues/723 now.

**Comment by ericwbailey** [comment]
[All set](https://github.com/a11yproject/a11yproject.com/commit/774929c6f3864acc9bd1d19316d538c93a590532)!
